### ISSUE TITLE
using import module

### ISSUE THREAD

**Issue by andy6804tw** [J-Chaniotis/external-ip#13]
Hi , I wanna replace require with import, but it didn't work for basic method. 

```js
const getIP = require('external-ip')(); 
         ↓
import getIP from 'external-ip';
```

thanks！

**Comment by N-Andronopoulos** [comment]
Why don't you use it like this:

```
import extIp from 'external-ip';

const getIp = extIp();

getIp((err, ip) => {
        if(err) {
                throw err;
        }
        console.log(ip);
});
```

**Comment by andy6804tw** [comment]
@N-Andronopoulos   ok, I got it  thank you~

**Comment by andy6804tw** [comment]
fixes #13
